docs/docs-content/clusters/edge/site-deployment/site-installation/edge-host-registration.md

```diff
@@ -8,13 +8,12 @@ tags: ["edge"]
 ---
 
 To use an Edge host with a host cluster, you must first register it with Palette. A registration token in the user data
-is required to complete the registration process. You have three options to register the Edge host with Palette.
+is required to complete the registration process. You have two options to register the Edge host with Palette.
 
-| **Method**          | **Description**                                                                                                                                                                                                                               | **Set up Effort** |
-| ------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
-| Auto Registration   | Edge hosts can automatically register with Palette by using a _Registration Token_. This method requires you to specify the registration token in the user data.                                                                              | Low               |
-| Manual Registration | You can manually enter a unique Edge host ID into Palette.                                                                                                                                                                                    | Low               |
-| QR Code             | Scan a QR code that takes you to a web application that registers the Edge host with Palette. This method is considered advanced with the benefit of simplifying the Edge host registration without needing a tenant token or a manual entry. | High              |
+| **Method**          | **Description**                                                                                                                                                  | **Set up Effort** |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| Auto Registration   | Edge hosts can automatically register with Palette by using a _Registration Token_. This method requires you to specify the registration token in the user data. | Low               |
+| Manual Registration | You can manually enter a unique Edge host ID into Palette.                                                                                                       | Low               |
```

Comment:
```suggestion
| Manual Registration | You can manually enter a unique Edge host ID in Palette.                                                                                                       | Low               |
```

Reply:
I'm a bit confused with the into, since it feels somehow very physical, like putting something into a container, you know? Would 'in' do the job?